src/hooks/Guilds/ipfs/usePinataIPFS.ts

```diff
@@ -0,0 +1,62 @@
+import { useTypedParams } from 'Modules/Guilds/Hooks/useTypedParams';
+import { useTranslation } from 'react-i18next';
+
+const DEFAULT_API_KEY =

```

Comment:
Are we hardcoding this?

Reply:
It was hardcoded in dxvote, that's why I put it here.

But I guess at the end of the day we're sending the API key in out post requests, so I'm not sure how effective it'll be to put it in a `.env`